README.md

```diff
@@ -289,6 +289,18 @@ dependencies, without committing, tagging, pushing or publishing.
 
 > Only update versions and dependencies; don't actually publish (this only affects publish).
 
+#### --registry [registry]
+
+```sh
+$ lerna publish --registry https://my-private-registry
```

Comment:
I'm not sure `publish` is the only command that would need this; certainly `bootstrap` could also potentially need it (for example, dependencies on private modules that aren't co-located in the monorepo).

We should also allow this setting in `lerna.json`, accessed via the `Command##getOptions()` API.